sockeye/output_handler.py

```diff
@@ -48,11 +50,15 @@ class OutputHandler:
     Abstract output handler interface
     """
 
-    def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.inference.TranslatorOutput):
+    def handle(self,
```

Comment:
why not add a member to translation output and set it in the translate CLI itself:
t_output.walltime = tic - toc